src/screens/application/container/applicationContainer.js

```diff
@@ -604,7 +599,6 @@ class ApplicationContainer extends Component {
 
     if (isConnected !== null && isConnected !== nextProps.isConnected && nextProps.isConnected) {
       this._fetchApp();
-      this.globalInterval = setInterval(this._refreshGlobalProps, 180000);
```

Comment:
it was global props refreshing, did you test it without this

Reply:
We do `_fetchApp()` which refreshes variables on startup right? So we don't need interval, everytime user restarts app, fetchApp() updates that data and it should be sufficient.

Reply:
yes i agree with you but we discuss to this before, if user not kill to application it will not refresh

Reply:
maybe we should update it on listener when app comes back from background to foreground, do we have any listener like that already? @mistikk